portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,45 @@
   width: 650px;
 }
 
+intro-title {
+    font-family: cursive;
+}
+
+greeting-button {
+  background-color: rgb(151, 91, 201);
+  border: none;
+  color: white;
+  padding: 15px ;
+  text-align: center;
+  font-size: 20px;
```

Comment:
I feel like I lost this comment too, but prefer keeping these CSS properties alphabetized.